include/abb_librws/v1_0/rw/cfg.h

```diff
@@ -0,0 +1,29 @@
+
+#include <abb_librws/rws.h>
+#include <abb_librws/v1_0/rws_client.h>
+
+#include <string>
+#include <iosfwd>
+
+namespace abb :: rws :: v1_0 :: rw
+{
+    using namespace rws::rw;
+}
+
+
+namespace abb :: rws :: v1_0 :: rw :: cfg
+{
+    /**
+     * \brief A function for loading configs to robot controller.
+     *
+     * https://developercenter.robotstudio.com/api/rwsApi/cfg_system_load_page.html
+     *
+     * \param client RWS client
+     *
+     * \return Current RAPID execution state.
+     *
+     * \throw \a RWSError if something goes wrong.
+     */
+    void loadCFGfile(RWSClient& client, const FileResource& resource);
```

Comment:
To be consistent with librws naming convention:
```suggestion
    void loadCFGFile(RWSClient& client, const FileResource& resource);
```